engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -33,35 +32,32 @@
 public final class SetViewportToSizeOf implements FBOManagerSubscriber, StateChange {
     private static SetViewportToSizeOf defaultInstance;
 
-    private BaseFBOsManager fboManager;
-    private ResourceUrn fboName;
+    private FBO fbo;
     private int fboWidth;
     private int fboHeight;
-    
-    @SuppressWarnings("FieldCanBeLocal")
-    private FBO fbo;
 
     /**
      * The constructor, to be used in the initialise method of a node.
      *
      * Sample use:
      *      addDesiredStateChange(new SetViewportToSizeOf("engine:sceneOpaque", displayResolutionDependentFboManager);
      *
-     * @param fboName a URN identifying an FBO.
-     * @param frameBuffersManager the BaseFBOsManager instance that will send change notifications via the update() method of this class.
+     * @param fbo
```

Comment:
Please check all javadocs in which an fbo parameter has been added: please make sure to add some description to each.